Movecraft/src/main/java/net/countercraft/movecraft/mapUpdater/update/CraftRotateCommand.java

```diff
@@ -118,27 +118,30 @@ public void doUpdate() {
                 validExterior.addAll(hitBox.difference(craft.getHitBox()));
             }
             //Check to see which locations in the from set are actually outside of the craft
+            SetHitBox visited = new SetHitBox();
             for (MovecraftLocation location : validExterior) {
                 if (craft.getHitBox().contains(location) || exterior.contains(location)) {
```

Comment:
Is `exterior.contains(location)` not redundant? `exterior` will now only be populated after the loop finishes, as `exterior.addAll(visited)` was moved outside of it.